app/elements/event-talk.mjs

```diff
@@ -0,0 +1,82 @@
+export default function EventTalk({ html, state = {} }) {
```

Comment:
I might've missed it. Where is this being used now?

Reply:
✅